frontend/src/components/elements/Favicon/Favicon.tsx

```diff
@@ -55,64 +57,24 @@ function extractEmoji(maybeEmoji: string): string {
   return ""
 }
 
-export interface Props {
-  element: ImmutableMap<string, any>
-}
-
-/**
- * Hidden element that overwrites the page's favicon with the provided image
- *
- * This has a complex lifecycle for emoji favicons, since it must first render
- * the emoji on the page to measure its dimensions, then hide it afterwards.
- */
-export const Favicon: React.FC<Props> = (props: Props) => {
-  const [render, setRender] = useState(true)
-  const [emoji, setEmoji] = useState("")
-  const [finalUrl, setFinalUrl] = useState("")
-  const textNode = useRef<SVGTextElement>(null)
-
-  // Re-render the entire component whenever the props are changed
-  // (aka user changed the favicon with st.beta_set_favicon)
-  useEffect(() => {
-    const url = props.element.get("url")
-    const emoji = extractEmoji(url)
-    if (emoji) {
-      setEmoji(emoji)
-      setRender(true)
+// We only need this one function of Twemoji to locate the CDN emoji image,
+// so we copy it instead of importing the whole library.
+// https://github.com/twitter/twemoji/blob/42f8843cb3aa1f9403d5479d7e3f7e01176ad08e/scripts/build.js#L571
```

Comment:
This is more of a preference thing but when copying other libraries, I would prefer to store them in a separate file/folder with their own licensing requirements. Looks like twemoji is fine with a mention in JS source so this works